File: backend/blueprints/about.py
"""
about.py -- Main Python blueprint for the About page that implements all functionality for the page
"""

from flask import Blueprint, jsonify

# Create about page blueprint
about = Blueprint('about', __name__, url_prefix='/api/about')

# Get about page information
@about.route('', methods=['GET'])
def get_about():
    about_info = {
        'title': 'Anonymous Bulletin Board',
        'description': 'An interactive, anonymous online community bulletin board where users can leave messages and interact with others freely. Everything expires in 24 hours.',
        'version': '1.0.0'
    }
    return jsonify(about_info)

File: frontend/src/pages/about/about.js
import React from 'react';
import { HomeButton, NavButtonBar } from '../../components';
import './about.css';

const About = () => {
  return (
    <div className="about-page">
      <h1>About the Bulletin Board</h1>
      <p>This is an interactive, anonymous online community bulletin board where users can leave messages and interact with others freely.</p>
      <p>Everything on the board expires after 24 hours, ensuring a fresh and comfortable space for interaction.</p>
      
      {/* Only include NavButtonBar, not HomeButton since this is the About page */}
      <NavButtonBar />
    </div>
  );
};

export default About;